err_stash.py

```diff
@@ -813,12 +824,12 @@ def delete_branch(self, msg, args):  # pragma: no cover
                     branches,
                 )
             )
-            settings["delete-branches"][branch_to_delete] = branches
-            self.save_user_settings(user, settings)
+            deleted_history[branch_to_delete] = branches
+            pickle.dump(deleted_history, file.open("wb"))
```

Comment:
Just make sure this is clearing the file and not appending objects to it.

Reply:
w: Opens in write-only mode. The pointer is placed at the beginning of the file and this will overwrite any existing file with the same name. It will create a new file if one with the same name doesn't exist.
wb: Opens a write-only file in binary mode.

I believe that they share the same behavior (beside text and binary mode). But i will test anyway